package.json

```diff
@@ -11,7 +11,7 @@
     "build": "ember build",
     "test": "ember try:testall"
   },
-  "repository": "https://github.com/dockyard/ember-validations",
+  "repository": "dockyard/ember-validations",
```

Comment:
The docs are a bit confusing, but this is the shortcut format [specified](https://docs.npmjs.com/files/package.json#repository) for github repos.